src/main/java/LogicManager.java

```diff
@@ -15,67 +23,136 @@ private boolean isIndexValidInt(String taskIndexString) {
             return false;
         }
     }
+    private Command processMarkUnmarkCommand(String[] words) {
+        if (words.length != 2) {
```

Comment:
Should we add a comment to explain how we got the number 2? eg "valid mark/unmark operations must have at least 2 words"